src/video-player.js

```diff
@@ -293,9 +334,21 @@ class VideoPlayer extends Utils.mixin(Eventable) {
     }
 
     this.videojs.fluid(bool);
+    this.videojs.trigger('fluid', bool);
     return this;
   }
 
+  changeSkin(skin) {
+    if (skin === 'dark' || skin === 'light') {
+      const prevSkin = this.videojs.options().skin;
+
+      this.videojs.removeClass(`cld-video-player-skin-${prevSkin}`);
+      this.videojs.addClass(`cld-video-player-skin-${skin}`);
+      this.videojs.options({ skin: skin });
+      this.videojs.trigger('themechange', { currSkin: skin, prevSkin: prevSkin });
```

Comment:
`trigger('skinchanged')` ?

Reply:
ill get rid of it for the mean time, lets deal with it when we will modify the css